recipe/meta.yaml

```diff
@@ -25,28 +28,29 @@ requirements:
   host:
     # Host dependencies are pinned to last build
     - python
-    - cython 3.0.8
+    - cython >=3.0.10
     - numpy {{ numpy }}
     - llvm-openmp 14.0.6  # [osx]
     - pip
-    - scipy >=1.6.0
-    - setuptools
-    - wheel
+    - scipy =1.9.3        # [py<312]
```

Comment:
generally, we omit the `=` before the version number. Should make no difference, it's just a style decision.